library/ssl_tls.c

```diff
@@ -7677,47 +7674,18 @@ int mbedtls_ssl_check_sig_hash( const mbedtls_ssl_context *ssl,
 {
     const int *cur;
 
-#if !defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
     if( ssl->conf->sig_hashes == NULL )
         return( -1 );
 
-    for ( cur = ssl->conf->sig_hashes; *cur != MBEDTLS_MD_NONE; cur++ )
+    /* MBEDTLS_MD_NONE (0x00) = SIGNATURE_NONE (0x00) */
+    for( cur = ssl->conf->sig_hashes; *cur != 0x00;  cur++ )
```

Comment:
Cf. API unification

Reply:
I clarified the use of the termination marker.